components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -310,6 +315,8 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
             <div>
                 <div style={styles.text}>Create a Kubeflow deployment</div>
 
+                <div style={styles.warning}>Warning: Kubeflow Deploy Site will be deprecated soon. </div>
```

Comment:
tiny nit: maybe 'This Kubeflow Deploy...' ? Just in case they're wondering if they're on the deploy site.

Reply:
@kunmingg Could you also add instructions point folks at the instructions for using the CLI? 
Thanks